upi/azure/deploy_azarm.sh

```diff
@@ -0,0 +1,17 @@
+echo "Start Deployment"
+az group deployment create \
+   --name $1 \
+   --resource-group $1 \
+   --template-uri "https://raw.githubusercontent.com/openshift/installer/master/upi/azure/azuredeploy.json" \
+     --parameters "runit.parameters.json"
+if [ $? -ne 0 ]
+then 
+   echo "Deployment Failed"
+   exit $?
+fi
+./openshift-install --dir=gw wait-for bootstrap-complete --log-level debug
```

Comment:
this expects the installer binary to exist in the current directory.. can we just remove this out of the script?

also why is the `--dir=gw` hard coded?

Reply:
Fixed.